pkg/kudoctl/kudoinit/prereq/webhook.go

```diff
@@ -253,9 +253,14 @@ func detectCertManagerCRD(extClient clientset.Interface, api certManagerVersion)
 	clog.V(4).Printf("Try to retrieve cert-manager CRD %s", testCRD)
 	crd, err := extClient.ApiextensionsV1beta1().CustomResourceDefinitions().Get(context.TODO(), testCRD, metav1.GetOptions{})
 	if err == nil {
-		// crd.Spec.Versions[0] must be the one that is stored and served, we should use that one
-		clog.V(4).Printf("Got CRD. Group: %s, Version: %s", api.group, crd.Spec.Versions[0].Name)
-		return api.group, crd.Spec.Versions[0].Name, nil
+		// Look through the versions and find the one that is the stored one
+		for _, v := range crd.Spec.Versions {
+			if v.Storage {
```

Comment:
Even if we don't support the `Storage` version we could still take one of the (smaller) `Served` version, correct? So the correct algorithm would be to pick the **highest** supported (`v.Storage || v.Served`) version that we find.